service/openapi/src/main/java/org/dotwebstack/framework/service/openapi/response/ResponseTemplateBuilder.java

```diff
@@ -132,9 +135,21 @@ private ResponseHeader mapHeader(Map.Entry<String, Header> headerEntry, String q
         .build();
   }
 
-  private boolean isDefault(MediaType content, Map<String, Object> extensions) {
-    return Objects.nonNull(extensions) && (boolean) content.getExtensions()
-        .get(DEFAULT_CONTENT_TYPE_VENDOR_EXTENSION);
+  private String getTemplateName(Map<String, Object> extensions) {
+    String templateName = null;
+    if (Objects.nonNull(extensions)) {
```

Comment:
Objects.nonNull alleen gebruiken binnen stream. 
```suggestion
    if (extensions != null) {
```

Reply:
Hebben we dit ook afgesproken voor DWS? Want dat zit hier namelijk wel vol mee...

Reply:
Eens. Code convention is 1 lijn. KDP == DWS. Het klopt dat dit nog niet overal is doorgevoerd.